docs/usage/configuration-options.md

```diff
@@ -3233,6 +3226,45 @@ It does not apply when you use a Personal Access Token as credential.
 When `platformCommit` is enabled, Renovate will create commits with GitHub's API instead of using `git` directly.
 This way Renovate can use GitHub's [Commit signing support for bots and other GitHub Apps](https://github.blog/2019-08-15-commit-signing-support-for-bots-and-other-github-apps/) feature.
 
+## platformOptions
```

Comment:
The latest build has test failures. I think you need to:

- Add more tests?
- Update the Jest snapshots?

Reply:
Yes, fixes otw